src/main/java/mate/controller/AddCarController.java

```diff
@@ -0,0 +1,40 @@
+package mate.controller;
+
+import java.io.File;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+public class AddCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private static final CarService carService =
+            (CarService) injector.getInstance(CarService.class);
+    private static final ManufacturerService manufacturerService =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/add/car.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws IOException {
+        Car car = new Car();
+        car.setModel(req.getParameter("model"));
+        Long id = Long.valueOf(req.getParameter("manufacturer"));
+        Manufacturer manufacturer = manufacturerService.get(id);
+        car.setManufacturer(manufacturer);
+        carService.create(car);
+        String tmp = req.getContextPath() + File.separator;
+        resp.sendRedirect(tmp);
```

Comment:
```suggestion
        resp.sendRedirect(req.getContextPath() + "/");
```
dont create redundant variables